node-sdk/src/lib.rs

```diff
@@ -30,13 +30,13 @@ impl UnleashEngine {
     #[wasm_bindgen(method, js_name = isEnabled)]
     pub fn is_enabled(&self, name: String, context: &JsValue) -> bool {
         let context: InnerContext = serde_wasm_bindgen::from_value(context.clone()).unwrap();
-        self.engine_state.is_enabled(&name, &context)
+        self.engine_state.is_enabled(&name, &context, None)
```

Comment:
The WASM compiles as part of the main build. This is just to not block the build